app/views/layouts/application.html.erb

```diff
@@ -33,13 +33,13 @@
       gtag("js", new Date());
       gtag("config", "UA-32848364-7D");
     </script>
-
     <script type="text/javascript" src="https://js.api.here.com/v3/3.1/mapsjs-core.js"></script>
     <script type="text/javascript" src="https://js.api.here.com/v3/3.1/mapsjs-service.js"></script>
+    <script type="text/javascript" src="https://js.api.here.com/v3/3.1/mapsjs-clustering.js"></script>
     <script type="text/javascript" src="https://js.api.here.com/v3/3.1/mapsjs-ui.js"></script>
     <script type="text/javascript" src="https://js.api.here.com/v3/3.1/mapsjs-mapevents.js"></script>
     <script type="text/javascript" >window.ENV_VARIABLE = 'https://<%= request.host %>'</script>
-    <script src='https://developer.here.com/javascript/src/iframeheight.js'></script>
+    <script type="text/javascript" src='https://developer.here.com/javascript/src/iframeheight.js'></script>
```

Comment:
NIT: Is it possible to use `async` or `defer` here to improve loading?

Reply:
I can add those & compare script loading times.